examples/version-migration/same-container/src/modelVersion1/inventoryList.ts

```diff
@@ -4,9 +4,10 @@
  */
 
 import { EventEmitter } from "@fluid-example/example-utils";
-import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct/internal";
+import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct/legacy";
+// eslint-disable-next-line import/no-internal-modules -- #26903: `cell` internals used in examples
```

Comment:
Just noting that as an example of migration, something we would expect external consumers would need to do, this use of internal/ APIs feels particularly egregious.

Reply:
FWIW, I was using "cell" before it became unpopular :-P

In some future update we can change over to a different DDS, the specific choice of DDS isn't core to the message of the example.